packages/create-block-interactive-template/block-templates/view.js.mustache

```diff
@@ -0,0 +1,48 @@
+{{#isBasicVariant}}
```

Comment:
I still can see double line breaks in this file. Is it ok?

Reply:
The idea behind these double-line breaks is to prepare this template for more variants.
As stated in `create-block-interactive-template/index.js` there's a variant called `basic` that is the one taken by default if no variant is specified.
But if we want to add some other variants, with these double-line breaks, this code will be more ready to add new ones (and more self-explanatory IMO). 
The code inside `{{#isBasicVariant}}` will not be executed when other variants are called and similar `{{#isXXXXXVariant}}` groups will need to be created for each new variant.

Reply:
I understand the purpose of the `{{isBasicVariant}}` syntax. What I don't understand are the double line breaks of [this particular file](https://github.com/WordPress/gutenberg/blob/5a165b17881958bd2ad53f7f868574346b73a054/packages/create-block-interactive-template/block-templates/view.js.mustache):

```js
store( {

	actions: {

		'{{namespace}}': {

			toggle: ( { context } ) => {

				context[ '{{namespace}}' ].isOpen = !context[ '{{namespace}}' ].isOpen;

			},

		},

	},
```

Instead of: 

```js
store( {
	actions: {
		'{{namespace}}': {
			toggle: ( { context } ) => {
				context[ '{{namespace}}' ].isOpen = !context[ '{{namespace}}' ].isOpen;
			},
		},
	},
```

Reply:
Oh, sorry, you're right. I wasn't noticing this.
Yes, there's no sense in these double-line breaks. 
I've removed them on a new commit to this PR 👍